modules/test/playwright/tests/object-web/main/objectFields.spec.ts

```diff
@@ -757,6 +757,62 @@ test.describe('Manage object fields through Model Builder', () => {
 });
 
 test.describe('Manage objectFields through Objects Admin UI', () => {
+	test("can update custom object field in a system object", async ({
+		apiHelpers,
+		objectFieldsPage,
+		page,
+	}) => {
+
+
+		const {items} = await apiHelpers.objectAdmin.getAllObjectDefinitions();
+
+		const systemObjectDefinition = items.find((item: ObjectDefinition) => {
+			return item.system === true;
+		});
+
+		const objectDefinitionField =
+			await apiHelpers.buildRestClient(ObjectFieldAPI);
+
+		const objectDefinitionFieldLabel = 'Custom Field'+getRandomInt();
```

Comment:
Rename to `objectFieldLabel` and add speces between the plus, so:

`const objectFieldLabel = 'objectFieldLabel'`

Reply:
Note that only the name property cannot be repeated, we can have several object fields in the same object definition with the same labels.